public/locales/en-US/page-meta-chain.json

```
@@ -0,0 +1,7 @@
+{
+  "heroBanner": {
+    "title": "THE UPGRADED ECOSYSTEM",
+    "subtitle": "Meta Chain: Build for Bitcoin & Ethereum",
+    "desc": "Unrivaled innovation and interoperability, without compromising on scalability or security. MetaChain is for building dApps on the world's largest blockchains."
```

are we separating into two words? 
```suggestion
    "desc": "Unrivaled innovation and interoperability, without compromising on scalability or security. Meta Chain is for building dApps on the world's largest blockchains."
```

---

let me check with the PM

---

ok, separated into two words!